frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -69,6 +69,7 @@ function ReaderCoptListener:onReadSettings(config)
 end
 
 function ReaderCoptListener:onConfigChange(option_name, option_value)
+    if option_name == "font_size" and option_value < 12 then return end
```

Comment:
I think all these need a comment.
`-- the option name font_size is historically and sadly shared by both reader modules, but fortunately their allowed values have different ranges, so we can distringuishthem`
and here:
`if option_name == "font_size" and option_value < 12 then return end -- aimed at Kopt`
and below `-- aimed at Cre`
(@Frenzie for a proper writing with less adjectives maybe ? :)

Reply:
If they're this simple to distinguish should we migrate them to their own values?

Reply:
But as for the comment, maybe shortened slightly like this?
```lua
-- font_size is historically and sadly shared by both filemanager and reader modules, but fortunately they can be distinguished by their different ranges
```